spaces/S000132/properties/P000066.md

```diff
@@ -0,0 +1,11 @@
+---
+space: S000132
+property: P000066
+value: false
+---
+
+Similar to the proof of {S28|P66}, it is easily seen for each finite sequence $t$, $\left[ t \right] := \left\{ x \in A \mid x \text{ extends } t \right\}$ is open in $A$.
+
+Now define $\mathcal U_n = \left\{ \left[ t \right] \mid t \in \omega^n \right\}$ and let any finite subcollections $\mathcal F_n \subseteq \mathcal U_n$, we can choose $x_n$ such that $\left[ \left< x_1, \dots, x_n \right> \right] \notin \mathcal F_n$ and $x_n > 2 x_{n - 1}$.
+
+Therefore $x \in A$ since $\delta(x) = 0$ and $x \notin \bigcup_{n < \omega} \mathcal F_n$.
```

Comment:
```suggestion
Similar to the proof of {S28|P66}, it is easily seen that for each finite sequence $t$, $\left[ t \right] := \left\{ x \in X \mid x \text{ extends } t \right\}$ is open in $X$.

Now define the open cover $\mathcal U_n = \left\{ \left[ t \right] \mid t \in \omega^n \right\}$.
Given any finite subcollections $\mathcal F_n \subseteq \mathcal U_n$, we can choose $x_n$ such that $\left[ \left< x_1, \dots, x_n \right> \right] \notin \mathcal F_n$ and $x_n > 2 x_{n - 1}$.

Then $x \in X$ since $\delta(x) = 0$ and $x \notin \bigcup_{n < \omega} \mathcal F_n$.
```